src/app/layout.tsx

```
@@ -23,6 +24,25 @@ export default function RootLayout({
           {children}
           <Footer />
         </Providers>
+        <Script
+          id="beusable-script"
+          dangerouslySetInnerHTML={{
+            __html: `
+            (function(w, d, a){
+                w.__beusablerumclient__ = {
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)